examples/.env.example

```
@@ -0,0 +1,27 @@
+BOT_TOKEN=
+BOT_PROXY=
+ADMIN_CHAT_ID=
+
+# База данных PostgreSQL
+# Для all-docker: DB_HOST=db
+# Для hybrid: DB_HOST=/var/run/postgresql (сокет) или localhost (TCP)
+DB_NAME=faunistica
+DB_HOST=db
+DB_PORT=5432
+DB_USER=faunistica
+DB_PASSWORD=faunistica
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_PORT key

(UnorderedKey)